### ISSUE TITLE
Before setting ibus, make this set particular xkb when there are more than 2 xkb

### ISSUE THREAD

**Issue by osamuaoki** [osamuaoki/inputmethod-shortcuts#1]
Case:  xkb=ru to ibus=mozc-jp  -- > broken

**Comment by Andrew15-5** [comment]
> Please open ticket [there](https://github.com/osamuaoki/inputmethod-shortcuts). [[1](https://github.com/ankostis/gnome-shell-quick-lang-switch/issues/7#issuecomment-1618247782)]

But you already have opened an issue (this one, #1). Should I open a new issue anyway?

**Comment by osamuaoki** [comment]
No need for new ticket.

Please test the next version which should address this issue.

Thanks for the idea.

**Comment by Andrew15-5** [comment]
I can't right now. I will post feedback here later.

**Comment by Andrew15-5** [comment]
I've checked and... and it's on a whole another level when compared to standard GNOME `Super+Space`! あざす！

I will note, though, that when switching really fast or when seeing and input field outline on focus, the switching between `us`/`jp` and `ru` is still noticeably slower, but the magnitude is much smaller (like 0.5 s). This shouldn't be a problem, as default behavior takes about 1-1.5 s before I can start typing. Now it takes about the same amount of time as I am switching fingers from shortcut position to "home row".

And also, because highlighting/outline still blinks during switching, focus-driven events will be triggered (AFAIK, GNOME always had this issue, but KDE doesn't). This is probably on GNOME, but I still think it's worth noting.

Great job! From now on, I will use this extension all the time.

P.S. When I updated the extension, and it broke, I had to wait until I have time to re-login/restart my machine. But when it broke, I already was used to new shortcuts. I tried to use them unconsciously, but they didn't work. This tells how easily and quickly I was able to integrate this extension in my workflow. Cool.
